### ISSUE TITLE
What is the meaning of 'model_id'? / Using this model in Colab

### ISSUE THREAD

**Issue by P-JuNYeonG** [thuml/Time-Series-Library#567]
Hello! I'm beginner in this area and trying to use TimesNet model to do time series forecast.
It's great for me your TimesNet_tutorial.ipynb. I've learned a great deal thanks to you.

I have some questions about TimesNet.

If it's not too much trouble, could you explain what the 'model_id' hyperparameter means? The default value of model_id is 'test' and 'ETTh1_96_96' in tutorial, but I want to train the model from scratch using my own data. I feel a little silly asking this, but I'm eager to use your model and I need to know this. And I want to train this model in Colab and save it for future predictions. Can you provide some guidance on this?

Thanks for helping me out, and I hope your day goes well!

**Comment by P-JuNYeonG** [comment]
Yeah now i know 'model_id' just a name of my model.. im so shamed :(

**Comment by wuhaixu2016** [comment]
Thank you for using TSlib. Good Luck~
